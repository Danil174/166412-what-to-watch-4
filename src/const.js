export const TextMovieRatings = {
  BAD: `Bad`,
  NORMAL: `Normal`,
  GOOD: `Good`,
  VERY_GOOD: `Very good`,
  AWESOME: `Awesome`,
};

export const SmallCardVideoSettings = {
  WIDTH: `100%`,
  HEIGHT: `100%`,
  IS_MUTED: true,
};

export const DEFAULT_GENRE = `All genres`;
export const MAX_GENRES_LENGTH = 9;
export const INITTIAL_LIST_LENGTH = 8;
export const SIMILAR_LIST_LENGTH = 4;

export const AuthorizationStatus = {
  AUTH: `AUTH`,
  NO_AUTH: `NO_AUTH`,
};

export const smallCardVideoStartTimeOut = 1000;

export const LoadErrorsTexts = {
  LIST_FAIL: `Не удалось загрузить список фильмов.`,
  PROMO_FAIL: `Не удалось загрузить промо фильм.`,
};

export const AppRoute = {
  LOGIN: `/login`,
  MOVIE_PAGE: `/movie_page`,
  FAVORITES: `/favorites`,
  ROOT: `/`,
};
