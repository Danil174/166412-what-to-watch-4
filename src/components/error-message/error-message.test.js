import React from "react";
import renderer from "react-test-renderer";
import ErrorMessage from "./error-message.jsx";
import {LoadErrorsTexts} from "../../const.js";

describe(`ErrorMessageTest`, () => {
  it(`render ErrorMessage`, () => {
    const tree = renderer
      .create(
          <ErrorMessage
            errorStatus={404}
            errorMessage={LoadErrorsTexts.LIST_FAIL}
          />
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
