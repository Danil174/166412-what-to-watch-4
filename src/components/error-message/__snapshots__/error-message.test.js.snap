// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessageTest render ErrorMessage 1`] = `
<div
  className="error"
>
  <p>
    Не удалось загрузить список фильмов.
  </p>
  <p>
    Обратитесь в техподдержку, код ошибки: 
    <strong>
      404
    </strong>
  </p>
</div>
`;
